.storybook/main.js

```diff
@@ -1,17 +1,4 @@
 module.exports = {
```

Comment:
Since you have removed this webpack reference can you do a quick depcheck and see if webpack is still required! Since rollup does the job buy storybook was using webpack internally, with this removed can you re-verify once!

Reply:
`❯ npx depcheck
npx: installed 148 in 13.492s
Unused devDependencies
* @storybook/addon-actions
* @storybook/addon-essentials
* @storybook/addon-links
* @types/jest
* @types/react-dom
* @types/react-router-dom
* babel-loader
* chromatic
* remark-cli
* remark-preset-lint-recommended
* storybook-react-router
* stylelint-config-standard
* stylelint-processor-styled-components
* ts-loader
Missing dependencies
* @visx/brush: ./src/lab/Graphs/LineAreaGraph/ComputationGraph.tsx
`

Reply:
I think ts-loader and chromatic won't be required anymore.
what is remark-cli?

Reply:
ok I think it's for mdx
https://github.com/remarkjs/remark

Reply:
what about storybook-react-router...is it used for the Links component...wasn't that component scraped?

Reply:
Yes. I don't think we require it now